webpack/assets/javascripts/react_app/components/AuditsList/ExpansiveView.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import DiffContainer from '../DiffView/DiffContainer';
 import { renderTemplatesDiff } from '../../../foreman_editor';
```

Comment:
isn't it redundant now?
also the `componentDidMount` callback with `renderTemplatesDiff` and the ref.